Automattic/sensei
# Add AI upsell button in quiz block and allow adding contents inside quiz block from other plugins

Update quiz block to allow injecting components, add upsell button above quiz block. This allows injecting quiz generation button from PRO.

## Proposed Changes

* Add slot in Quiz block
* Add script to inject upsell button in Quiz block

## Testing Instructions

1. Create a lesson
2. Add a quiz
3. Make sure the quiz block has a button on top with disabled text inside and Pro badge
4. Make sure clicking the button takes you to the pro page in senseilms.com

## Video/Screenshots

https://user-images.githubusercontent.com/6820724/231333939-e2fa72a7-5161-4f62-a9cc-705c5c1d4799.mov

## New/Updated Hooks

`sensei_ai_quiz_generation_available`: Check if quiz generation feature is available.

## Pre-Merge Checklist
